### DISCUSSION TITLE
Faster alternative to `archivemount`

### DISCUSSION THREAD

**Discussion by musjj** [jarun/nnn#1627] [Ideas]
`archivemount` is pretty slow for big archives, so it would be nice if the user can choose other options.
It's probably not complete, but I've collected some alternatives that is probably faster:

- https://github.com/mxmlnkn/ratarmount
- https://github.com/google/fuse-archive
- https://sourceforge.net/projects/avf/

Reference: https://unix.stackexchange.com/questions/24032/faster-alternative-to-archivemount

**Comment by jarun** [comment]
`avfs` doesn't support too many formats though it's available in the ubuntu repo. And it has other deps.
The other 2 are not available in the repo.

`ratarmount` generates a local cache file.

`fuse-archive` looks promising though.

I am considering adding a config option to make this flexible so that users can pick their favourite utility.

**Comment by jarun** [comment]
Added new config option `NNN_ARCHMNT` to specify the archive mounter utility.
`archivemount` remains the default.

**Reply by musjj** [reply]
Thanks, works pretty well so far!
